executor/admin_test.go

```diff
@@ -841,89 +917,93 @@ func (s *testSuite3) TestAdminCheckPartitionTableFailed(c *C) {
 	for i := 0; i <= 5; i++ {
 		partitionIdx := i % len(tblInfo.GetPartitionInfo().Definitions)
 		indexOpr := tables.NewIndex(tblInfo.GetPartitionInfo().Definitions[partitionIdx].ID, tblInfo, idxInfo)
-		txn, err := s.store.Begin()
-		c.Assert(err, IsNil)
-		_, err = indexOpr.Create(s.ctx, txn, types.MakeDatums(i+8), kv.IntHandle(i+8), nil)
-		c.Assert(err, IsNil)
+		txn, err := store.Begin()
+		require.NoError(t, err)
+		_, err = indexOpr.Create(ctx, txn, types.MakeDatums(i+8), kv.IntHandle(i+8), nil)
+		require.NoError(t, err)
 		err = txn.Commit(context.Background())
-		c.Assert(err, IsNil)
+		require.NoError(t, err)
 		err = tk.ExecToErr("admin check table admin_test_p")
-		c.Assert(err, NotNil)
-		c.Assert(err.Error(), Equals, fmt.Sprintf("[executor:8133]handle %d, index:types.Datum{k:0x1, decimal:0x0, length:0x0, i:%d, collation:\"\", b:[]uint8(nil), x:interface {}(nil)} != record:<nil>", i+8, i+8))
+		require.Error(t, err)
+		require.EqualError(t, err, fmt.Sprintf("[executor:8133]handle %d, index:types.Datum{k:0x1, decimal:0x0, length:0x0, i:%d, collation:\"\", b:[]uint8(nil), x:interface {}(nil)} != record:<nil>", i+8, i+8))
```

Comment:
```suggestion
		require.EqualError(t, err, fmt.Sprintf("[executor:8133]handle %d, index:types.Datum{k:0x1, decimal:0x0, length:0x0, i:%d, collation:\"\", b:[]uint8(nil), x:interface {}(nil)} != record:<nil>", i+8, i+8))
```

can be reduced. ditto if others.

Reply:
already fix, but the result of check_dev_2  is unstable, failed several times because of other tests.